jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/Queue.kt

```diff
@@ -0,0 +1,11 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.sqs
+
+class Queue(queueUrl: String) {
+    val regionId = queueUrl.substringAfter("sqs.").substringBefore('.')
+    val accountId = queueUrl.substringAfter("amazonaws.com/").substringBefore('/')
+    val queueName = queueUrl.substringAfter("$accountId/")
```

Comment:
We should throw an exception if any of these are empty (or fail in some way). It would also be good to add tests for the functions along with giving them China and Govcloud urls